sensing/launch_detection.c

```diff
@@ -0,0 +1,119 @@
+/*******************************************************************************
+ * Copyright (c) 2009-2014, MAV'RIC Development Team
+ * All rights reserved.
+ * 
+ * Redistribution and use in source and binary forms, with or without 
+ * modification, are permitted provided that the following conditions are met:
+ * 
+ * 1. Redistributions of source code must retain the above copyright notice, 
+ * this list of conditions and the following disclaimer.
+ * 
+ * 2. Redistributions in binary form must reproduce the above copyright notice, 
+ * this list of conditions and the following disclaimer in the documentation 
+ * and/or other materials provided with the distribution.
+ * 
+ * 3. Neither the name of the copyright holder nor the names of its contributors
+ * may be used to endorse or promote products derived from this software without
+ * specific prior written permission.
+ * 
+ * THIS SOFTWARE IS PROVIDED BY THE COPYRIGHT HOLDERS AND CONTRIBUTORS "AS IS" 
+ * AND ANY EXPRESS OR IMPLIED WARRANTIES, INCLUDING, BUT NOT LIMITED TO, THE 
+ * IMPLIED WARRANTIES OF MERCHANTABILITY AND FITNESS FOR A PARTICULAR PURPOSE 
+ * ARE DISCLAIMED. IN NO EVENT SHALL THE COPYRIGHT HOLDER OR CONTRIBUTORS BE 
+ * LIABLE FOR ANY DIRECT, INDIRECT, INCIDENTAL, SPECIAL, EXEMPLARY, OR 
+ * CONSEQUENTIAL DAMAGES (INCLUDING, BUT NOT LIMITED TO, PROCUREMENT OF 
+ * SUBSTITUTE GOODS OR SERVICES; LOSS OF USE, DATA, OR PROFITS; OR BUSINESS 
+ * INTERRUPTION) HOWEVER CAUSED AND ON ANY THEORY OF LIABILITY, WHETHER IN 
+ * CONTRACT, STRICT LIABILITY, OR TORT (INCLUDING NEGLIGENCE OR OTHERWISE) 
+ * ARISING IN ANY WAY OUT OF THE USE OF THIS SOFTWARE, EVEN IF ADVISED OF THE 
+ * POSSIBILITY OF SUCH DAMAGE.
+ ******************************************************************************/
+
+/*******************************************************************************
+ * \file launch_detection.c
+ * 
+ * \author MAV'RIC Team
+ * \author Dylan Bourgeois
+ *   
+ * \brief Detect launches, used in the implementation of paper :
+ *
+ *		Automatic Re-Initialization and Failure Recovery for Aggressive
+ *		Flight with a Monocular Vision-Based Quadrotor
+ *		M. Faessler, F. Fontana, C. Forster, D. Scaramuzza
+ *		IEEE International Conference on Robotics and Automation (ICRA), Seattle, 2015.
+ * 		http://rpg.ifi.uzh.ch/docs/ICRA15_Faessler.pdf
+ *
+ ******************************************************************************/
+
+#include <stdlib.h>
+
+#include "launch_detection.h"
+#include "vectors.h"
+
+
+#define ACC_X acc[0]		//< Alias for X acceleration
```

Comment:
I think we are missing one '/' for the comment to be properly reported in the doc
isn't it @jlecoeur ?